usr/src/lib/libnsl/common/mapfile-vers

```diff
@@ -39,6 +40,11 @@
 
 $mapfile_version 2
 
+SYMBOL_VERSION ILLUMOS_0.1 {
+    global:
+        svc_create_port;
```

Comment:
Do you intend to have this be a new public interface? I suspect not (based at least on lack of manual page updates). You should add this to the appropriate private symbol group which in this case is SUNWprivate_1.5.